public/css/global.css

```diff
@@ -172,8 +172,39 @@ header .icon .text {
 
 .notifications {
     color: #000000;
-    min-width: 450px!important;
-    font-size: 15px;
+    min-width: 650px!important;
```

Comment:
Je sais mais ça marche comme ça et Sckypzyck nous a dit de ne pas nous attarder sur le viseuls alors je push comme ça.
Mais j'ai noté qu'il faudrait faire plus propore si possible.